classes/characters/Ambassador.py

```diff
@@ -5,5 +5,8 @@ class Ambassador(Character):
     def __init__(self):
         pass        
     
+    def description(self) -> str:
+        return "Ambassadors can do the following:\n    Exchange: Exchange cards with the Court. First take 2 random cards from the Court deck. Choose which, if any, to exchange with your face-down cards. Then return two cards to the Court deck. \n    Block: Steal"
```

Comment:
instead of 2, use the dynamic value from config

Reply:
but nbd

Reply:
ah yes we should include that, i copy/pasted from the coup site